code/modules/materials/materials.dm

```diff
@@ -346,7 +346,7 @@ var/list/name_to_material
 	return round(totalPhoron/100)
 */
 
-/material/stone

```

Comment:
As discussed.
This needs to be changed back to /material/stone and the path in material_recipes needs to be changed from /material/sandstone to /material/stone

Reply:
Why is that? I could see us adding a proper "stone" material in the NBT.

Reply:
Because there is a /material/stone/marble
By changing the /material/stone to /material/sandstone the inheritance chain is broken.